docs/source/howto/share_data.rst

```diff
@@ -0,0 +1,248 @@
+tell AiiDA the **final results** you would like to be reproducible, and AiiDA will automatically include their entire provenance using the :ref:`topics:provenance:consistency:traversal-rules`.
+
+Exporting individual nodes
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Let's say the key results of your study are contained in three AiiDA nodes with PKs ``12``, ``123``, ``1234``.
+Exporting those results together with their provenance is as easy as:
+
+.. code-block:: console
+
+    $ verdi export create my-calculations.aiida --nodes 12 123 1234
+
+As usual, you can use any identifier (label, PK or UUID) to specify the nodes to be exported.
+
+The resulting archive file ``my-calculations.aiida`` contains all information pertaining to the exported nodes.
+The default traversal rules make sure to include the complete provenance of any node specified and should be sufficient for most cases.
+See ``verdi export create -h`` for ways to modify the traversal rules.
+
+.. tip::
+
+    If you want to make sure that the archive includes everything you intended, you can create a new profile and import it:
+
+    .. code-block:: console
+
+            verdi quicksetup --profile test-export  # create new profile
+            verdi --profile test-export import my-calculations.aiida
+
+            # now use e.g. the QueryBuilder to query the database
+
+Please remember to use **UUIDs** when pointing your colleagues to data *inside* an AiiDA export file, since UUIDs are guaranteed to be universally unique (while PKs aren't).
+
+Using groups for data exporting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the number of results to be exported is large, for example in a high-throughput study, use the ``QueryBuilder`` to add the corresponding nodes to a group ``my-results`` (see :ref:`how-to:data:organize:group`).
+Then export the group:
+
+.. code-block:: console
+
+    $ verdi export create my-calculations.aiida --groups my-results
+
+Publishing AiiDA export files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+AiiDA export files can be published on any research data repository, for example the `Materials Cloud Archive`_, `Zenodo`_, or the `Open Science Framework`_.
+When publishing AiiDA export files on the `Materials Cloud Archive`_, you also get an interactive *EXPLORE* section, which allows peers to browse the AiiDA provenance graph directly in the browser.
+
+
+.. _Zenodo: https://zenodo.org
+.. _Open Science Framework: https://osf.io
+.. _Materials Cloud Archive: https://archive.materialscloud.org
+
+
+Importing an archive
+^^^^^^^^^^^^^^^^^^^^
+
+Use ``verdi import`` to import AiiDA archives into your current AiiDA profile.
+``verdi import`` accepts URLs, e.g.:
+
+.. code-block:: console
+
+    $ verdi import "https://archive.materialscloud.org/record/file?file_id=2a59c9e7-9752-47a8-8f0e-79bcdb06842c&filename=SSSP_1.1_PBE_efficiency.aiida&record_id=23"
+
+During import, AiiDA will avoid identifier collisions and node duplication based on UUIDs (and email comparisons for ``User`` entries).
```

Comment:
Maybe not necessary, since the entries will not have anything directly to do with the Python representation of the same.
```suggestion
During import, AiiDA will avoid identifier collisions and node duplication based on UUIDs (and email comparisons for :py:class:~`aiida.orm.users.User` entries).
```